bot.py

```diff
@@ -45,11 +45,27 @@ def build_response_text(rate, bank):
     )
         
 def build_response_image(text):
```

Comment:
Please wrap this functionality into a class. 
It is better to have a class with ability to customize image parameters

Reply:
Done